packages/open-payments/README.md

```diff
@@ -0,0 +1,249 @@
+
+### `UnauthenticatedClient`
+
+This client allows making requests to access publicly available resources, without needing authentication.
+The three available resources are [Payment Pointers](https://docs.openpayments.guide/reference/get-payment-pointer), [Payment Pointer Keys](https://docs.openpayments.guide/reference/get-payment-pointer-keys), and [ILP Stream Connections](https://docs.openpayments.guide/reference/get-ilp-stream-connection).
+
+```ts
+import { createUnauthenticatedClient } from '@interledger/open-payments'
+
+const client = await createUnauthenticatedClient({
+  requestTimeoutMs: 1000, // optional, defaults to 5000
+  logger: customLoggerInstance // optional, defaults to pino logger
+})
+
+const paymentPointer = await client.paymentPointer.get({
+  url: 'https://cloud-nine-wallet/alice'
+})
+```
+
+### `AuthenticatedClient`
+
+This client provides methods to make requests to all of the Open Payment APIs, including both the authorizaton and resource specs. Each request requiring authentication will be signed (using [HTTP signatures](../../docs/architecture.md#http-signature-utils)) with the given private key.
+
+```ts
+import { createAuthenticatedClient } from '@interledger/open-payments'
+
+const client = await createAuthenticatedClient({
+  keyId: KEY_ID,
+  privateKey: PRIVATE_KEY,
+  paymentPointerUrl: PAYMENT_POINTER_URL
+})
+```
+
+In order to create the client, three properties need to be provided: `keyId`, the `privateKey` and the `paymentPointerUrl`:
+
+| Variable            | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `paymentPointerUrl` | The valid payment pointer with which the client making requests will identify itself. A JSON Web Key Set document, including the public key that the client instance will use to protect requests and any user-facing information about the client instance used in interactions, MUST be available at the `{paymentPointerUrl}/jwks.json` url. This will be used as the `client` field during [Grant Creation](https://docs.openpayments.guide/reference/post-request). |
```

Comment:
`any user-facing information about the client instance used in interactions` would be on the payment pointer itself (as opposed to `/jwks.json`)